spec/lib/authorities/linked_data/search_config_spec.rb

```diff
@@ -230,14 +231,14 @@
     end
 
     context 'when label specified as ldpath' do
-      before { allow(full_config).to receive(:results).and_return(ldpath_results_config) }
       it 'returns the ldpath' do
-        expect(full_config.results_label_ldpath).to eq 'skos:prefLabel ::xsd:string'
+        expect(full_config.results_label_ldpath).to eq 'skos:prefLabel'
```

Comment:
The ldpaths in loc_full_config fixture do not have ::xsd:string on the ldpaths.  It is optional.  The ldpath service already tests paths with and without the type (not in this PR).